### ISSUE TITLE
NVDA reporting uneeded information when loding IE 10

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2624]
**Reported by Brendon22 on 2012-08-29 07:36**
Hello,
When loding a website in IE 9 Version 9.0.8112.16421 on my computer for the first time. While IE is loding NVDA will say Paine and unknown. And also NVDA will say unknown when returning to your desktop.
I think this is unneeded information.
Using Windows 7 64-Bit

**Comment by bhavyashah** [comment]
I can reproduce the issue of 'unknown' being reported when a program is launched reliably via the following STR with NVDA version 2017.2 and Windows 8.1:
* Open the Start Menu.
* Search for any program, let's say Microsoft Word, or Internet Explorer, or Teamtalk, all three of which I tested at my end. Once the program appears in the search results, press it.
Expected Result: NVDA should announce the program's window title when it launches.
Actual Result: Before anything is spoken, NVDA says 'unknown' once. After that one 'unknown', NVDA works as expected.

**Comment by josephsl** [comment]
Hi, reproducible on Windows 10 as well, may need to debug this via event tracking to find out what’s up. Thanks.

**Comment by Brian1Gaff** [comment]
I think it misses the event telling it that the box contains the words you 
are wanting to read for some reason.
 Its one of those irritating things I see all over the place and I've always 
put it down to the screenreader not keeping up.

I often get it on the first launch of Firefox as well, when the processor 
usage goes berserk.


Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Adriani90** [comment]
there are several issues about this. @FelixGruetzmacher reported recently an issue. @josephsl do you have any other suggestions? @leonardder I think you were also involved in discussions dealing with this. Any thoughts?

**Comment by FelixGruetzmacher** [comment]
Hello,
what I reported, and am still seeing, is what I call unreliable focus
tracking. Especially after starting an application, it doesn't always
register with NVDA that focus is now on a specific control inside the new
application's window. NVDA variously assumes focus is still on the desktop,
or on some other object inside the new window, or on the new window itself.
Most often encountered with Outlook, Chrome, and some development
environments, this bug often necessitates some alt+tabbing or similar
action to get back in sync.
As far as redundancy of output is concerned, there is diffusion of
responsibility as the user experience is often shaped by at least two
independently developed components, namely, the screen reader and the
browser or other app concerned. There is no single group or individual
responsible for designing these user experiences, so we are and will always
be one translation step away from sighted users. The obvious solution of
more strictly formalizing accessibility interfaces would not go down well
with screen reader manufacturers as it directly threatens some of their
vital unique selling points, and might not even be viable as many app
developers feel no motivation to use these interfaces at all, and those who
do might feel even less motivated if the number of hoops they need to jump
through is increased.
Best,
Felix

Am Mi., 7. Nov. 2018 um 19:05 Uhr schrieb Adriani90 <
notifications@github.com>:

**Comment by Adriani90** [comment]
@FelixGruetzmacher     could you please test the last Alpha and see if it still occurs often?

**Comment by Adriani90** [comment]
I am closing this in favor of #8389 because this issue is very general and occurs in several situations regardless of the application. The discussion in that issue goes further and will continue there.
